snmp/tests/compose/data/juniper-ex.snmprec

```diff
@@ -1,4 +1,4 @@
-1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2636.1.1.1.2.30
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2636.1.1.1.4.30.2
```

Comment:
I think we might want to test both cases at lest once. E.g. 

- `snmp/tests/compose/data/juniper-ex.snmprec` with `1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2636.1.1.1.2.30`
- `snmp/tests/compose/data/juniper-ex-variation.snmprec` with `1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2636.1.1.1.4.30.2`

Same comment for mx and srx

Reply:
Ok!